custom_components/tuya_local/devices/pool_thermometer.yaml

```diff
@@ -0,0 +1,36 @@
+name: Raddy PT-5 pool thermometer
+products:
+  - id: qirx90k6hg4qzpbz
+    model: EM3406B-TY
+entities:
+  - entity: sensor
+    name: indoor temperature
+    class: temperature
+    dps:
+      - id: 101
+        name: sensor
+        type: integer
+        class: measurement
+        unit: C
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: indoor humidity
+    class: humidity
+    dps:
+      - id: 102
+        name: sensor
+        type: integer
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: pool temperature
```

Comment:
As this is the primary function of the device, I suggest to remove the name and allow HA to use the translations for temperature class.